help/en/html/tools/Reporters.shtml

```diff
@@ -23,33 +53,185 @@
+        using a <a href="../tools/logixng/reference/chapter5.shtml#digital_action_item">LogixNG</a>
+        or via a <a href="scripting/index.shtml">script</a>.
+        
+        <h2>The Reporter Table</h2>
+
+        <p>The <a href="../../package/jmri/jmrit/beantable/ReporterTable.shtml">Reporter Table</a>
+        provides easy access to the contents of all Reporters that JMRI knows about. 
+        JMRI automatically creates Reporter objects for certain connection types
+        (currently <a href="../hardware/loconet/Addressing.shtml#transponding">LocoNet 
+        transponders</a> and <a href="../hardware/rfid/index.shtml">RFID readers</a>.
+        There is also an <a href="../../package/jmri/jmrit/beantable/ReporterAddEdit.shtml">"Add..." button</a>
+        for creating your own Reporters, either "internal" or for any 
+        <a href="https://www.jmri.org/help/en/html/hardware/mqtt/index.shtml#objects">MQTT connections</a>.</p>
+
+        <h2>Creating and Updating Reporters</h2>
 
-        <p>The initial implementation is for LocoNet (Digitrax) transponding, where the Reporter
-        provides access to information from Digitrax <a href=
+        <p>Reporters can be created manually by direct entry or via script into the Reporter Table. For some hardware
+        reporters, Reporters are automatically created in the Reporter Table.  Current Report is populated and cleared by some 
+        implementations (including being cleared when the same reported object appears in another Reporter).
+        For example, for an RFID Reporter, if an ID tag is seen at second reporter, it will
+        be removed from the first reporter.  For other types of Reporters (e.g. Internal REporters), this 
+        must be done manually or via a script. </p>
+        
+        <p>Last Report is always set equal to the Current Report and is only changed by a new (non-blank)
+        Current Report.  This is to handle the situation where a report may not be able to sent a timely report
+        but it is useful to know the last report received.  [Advanced: See the <a href="../../../../JavaDoc/doc/jmri/Reporter.html">
+        Java documentation</a></p>
+
+        <p>The following table provides details for implementations as of JMRI Release 5.3.7 [Asterisk (*) indicates 
+        additional information below the table.  Number sign (#) indicates information that had not been verified when
+        this page was updated.] [Advanced Reference Info: Code Source is the java program defining
+        this type of Reporter. See <a href="../../../../JavaDoc/doc/jmri/doc-files/Reporter.png">this diagram.
+        </a>]</p>        
+       
+        <table class="boxed">
+        <tr style="text-align: left">
+          <th>Source</th>
+          <th>When Reporter Created</th>
+          <th>Current Report Populated</th>
+          <th>Current Report Cleared</th>
+          <th>Add'l Table</th>
+          <th>Code Source</th>
```

Comment:
Would it be better to make the Code Source column into a link to the Javadoc for the class?  That contains all this info in a less compressed format.